org.civicrm.hremergency/api/v3/Contact/Deleteemergencycontact.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+/**
+ * Contact.DeleteEmergencyContact API specification
+ *
+ * @param array $spec description of fields supported by this API call
+ * @return void
+ */
+function _civicrm_api3_contact_deleteemergencycontact_spec(&$spec) {
+  $spec['id']['api.required'] = 1;
```

Comment:
Always try to use the complete specification as possible. Please also add name, title, description and type

Reply:
I've added description, type and title. I don't think name is necessary since it defaults to the array key.